context/AuthHelper.tsx

```diff
@@ -14,6 +14,12 @@ type UserCredentials = {
   password: string;
 };
 
+type logoutType = {
+  setIsSignedIn: (bool: false) => void;
```

Comment:
Could you please explain what `setIsSignedIn` is used for?

Reply:
its a boolean that is set in the login page to determine if a user is logged in. In the logoutFunction we are setting it to false by default

Reply:
Ah, got it, thank you for explaining!